README.md

```diff
@@ -28,9 +28,11 @@ Simply check out this repo and run the appropriate command below :rocket:
 ### In Sauce Cloud
 
 ```shell
-saucectl run
+saucectl run --env APP=<your-app-ipa-file> --env TEST_APP=<your-test-app-ipa-file>
```

Comment:
The `--env` flag is meant for specifying environment variables on the target device.